src/sentry/profiles/profile_chunks.py

```diff
@@ -0,0 +1,41 @@
+from snuba_sdk import Column, Condition, Direction, Op, OrderBy, Query, Request, Storage
+
+from sentry import options
+from sentry.search.events.types import ParamsType
+from sentry.snuba.dataset import Dataset, StorageKey
+from sentry.snuba.referrer import Referrer
+from sentry.utils.snuba import raw_snql_query
+
+
+def get_chunk_ids(
+    params: ParamsType,
+    profiler_id: str,
+    project_id: int,
+) -> list[dict[str, str]]:
+    max_chunks = options.get("profiling.continuous-profiling.chunks-set.size")
+    query = Query(
+        match=Storage(StorageKey.ProfileChunks.value),
+        select=[
+            Column("chunk_id"),
+        ],
+        where=[
+            Condition(Column("start_timestamp"), Op.GTE, params.get("start")),
+            Condition(Column("end_timestamp"), Op.LT, params.get("end")),
+            Condition(Column("project_id"), Op.EQ, project_id),
+            Condition(Column("profiler_id"), Op.EQ, profiler_id),
+        ],
+        orderby=[OrderBy(Column("start_timestamp"), Direction.DESC)],
+    ).set_limit(max_chunks)
```

Comment:
This feels like it should be a paginated API. So for a given project id, profiler id, start/end timestamp, the frontend should be able to request the first page, then load more as needed not restricted to the first 50 results.

Reply:
@Zylphrex yes, indeed in the PR description I mentioned this:

> In later iterations, if necessary, we might provide a pagination mechanism so that we'll be able to scroll the whole profile directly from the flamechart page itself.

I had discussed this offline with @phacops and we decided to keep things simple and start with a _fixed size_ solution for the first iteration, but providing pagination is definitely in the plans.